docs/index.rst

```diff
@@ -3,185 +3,287 @@ The confluent_kafka API
 
 A reliable, performant and feature rich Python client for Apache Kafka v0.8 and above.
 
-Clients
+Client API 
+   - :ref:`AdminClient <pythonclient_adminclient>`
+   - :ref:`AvroProducer <avro_producer>`
+   - :ref:`AvroConsumer <avro_consumer>`
    - :ref:`Consumer <pythonclient_consumer>`
+   - :ref:`DeserializingConsumer <serde_consumer>`
    - :ref:`Producer <pythonclient_producer>`
-   - :ref:`AdminClient <pythonclient_adminclient>`
+   - :ref:`SerializingConsumer <serde_producer>`
+   - :ref:`SchemaRegistry <schemaregistry_client>`
+
+Serialization API
+    - :ref:`Deserializer <serde_deserializer>`
+    - :ref:`AvroDeserializer <schemaregistry_avro_deserializer>`
+    - :ref:`JSONDeserializer <schemaregistry_json_deserializer>`
+    - :ref:`ProtobufDeserializer <schemaregistry_protobuf_deserializer>`
+
+    - :ref:`Serializer <serde_Serializer>`
+    - :ref:`AvroSerializer <schemaregistry_avro_serializer>`
+    - :ref:`JSONSerializer <schemaregistry_json_serializer>`
+    - :ref:`ProtobufSerializer <schemaregistry_protobuf_serializer>`
+
+:ref:`Transactional Producer API <pythonclient_transactional>`
 
+:ref:`Kafka Client Configuration <pythonclient_configuration>`
 
 Supporting classes
     - :ref:`Message <pythonclient_message>`
     - :ref:`TopicPartition <pythonclient_topicpartition>`
+    - :ref:`SerializationContext <serde_ctx>`
+    - :ref:`Schema <schemaregistry_schema>`
+    - :ref:`RegisteredSchema <schemaregistry_registered_schema>`
+    - :ref:`SchemaRegistryError <schemaregistry_error>`
     - :ref:`KafkaError <pythonclient_kafkaerror>`
     - :ref:`KafkaException <pythonclient_kafkaexception>`
+    - :ref:`ConsumeError <serde_consumer_error>`
```

Comment:
It probably already does, but the ConsumeError should reside under confluent_kafka(.error?) and not under confluent_kafka.serializers.

Reply:
yes this is an artifact of when I had SerializingProducer/DeserializingConsumer under serialization in the TOC. Will fix

Reply:
The ref tag that is *